libafl_bolts/src/llmp.rs

```diff
@@ -788,7 +782,7 @@ pub struct LlmpPage {
     /// Set to != 1 by the receiver, once it got mapped.
     /// It's not safe for the sender to unmap this page before
     /// (The os may have tidied up the memory when the receiver starts to map)
-    pub safe_to_unmap: AtomicU16,
+    pub readers: AtomicU16,
```

Comment:
maybe `reader_count` to make it clear it's a number (not that the type wouldn't say that already, but..)